etc/dev-start.sh

```diff
@@ -19,6 +19,9 @@ export INBUCKET_STORAGE_PARAMS="path:/tmp/inbucket"
 export INBUCKET_STORAGE_RETENTIONPERIOD="3h"
 export INBUCKET_STORAGE_MAILBOXMSGCAP="300"
 
+
+export INBUCKET_SMTP_PROXYHOSTS="127.0.0.1"
```

Comment:
Let's move this up alphabetically with the other SMTP options, and leave commented out by default.

Reply:
agreed